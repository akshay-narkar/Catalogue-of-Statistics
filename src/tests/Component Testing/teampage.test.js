/*eslint-disable*/

import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import React from 'react';
import Teampage from '../../Containers/teampage';
import store from './store';

const props = {
  calldispatch: () => {},
  teaminfo: [{ id: 1 }],
};

it('UI is unaltered wrt first snapshot', () => {
  const elem = renderer.create(<BrowserRouter><Provider store={store}><Teampage props={props} /></Provider></BrowserRouter>).toJSON();

  expect(elem).toMatchSnapshot();
});
