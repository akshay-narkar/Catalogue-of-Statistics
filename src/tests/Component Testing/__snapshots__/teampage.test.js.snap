// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI is unaltered wrt first snapshot 1`] = `
Array [
  <div
    className="container"
  >
    <a
      className="link"
      href="/"
      onClick={[Function]}
    >
      <button
        className="my-2 p-2 btn btn-primary btn-sm"
        type="button"
      >
        Home
      </button>
    </a>
  </div>,
  <h1
    className="text-center mb-3 pb-3"
  >
     Team Details 
  </h1>,
  <div
    className=""
  >
    <div
      className="container m-auto text-center border"
    >
      Loading... (API might take time. Only 10 Requests/Min)
    </div>
  </div>,
]
`;
